maliput/tests/lane_test.rs

```diff
@@ -76,3 +76,22 @@ fn lane_api() {
     let expected_segment_id = String::from("0_0");
     assert_eq!(segment.id(), expected_segment_id);
 }
+
+#[test]
+fn lane_end_test() {
+    let road_network = common::create_t_shape_road_network();
+    let road_geometry = road_network.road_geometry();
+
+    let lane_id = String::from("0_0_1");
+    let lane = road_geometry.get_lane(&lane_id);
+    let lane_end_start = maliput::api::LaneEnd::Start(&lane);
+    match lane_end_start {
+        maliput::api::LaneEnd::Start(lane) => assert_eq!(lane.id(), lane_id),
```

Comment:
Should use another name for `lane` to avoid alising. Same below.

Reply:
Yeah makes sense. I am changing a bit this test in a follow-up PR. Let me address it in the next PR.

Reply:
https://github.com/maliput/maliput-rs/pull/62